src/sass/card/card.scss

```diff
@@ -0,0 +1,137 @@
+@import "../mixins/private/bubble-mixins";
+@import "../mixins/private/token-mixins";
+@import "../variables/variables";
+
+.card {
+    display: flex;
+    flex-direction: column;
+    width: $card-size-default;
+
+    &--large {
```

Comment:
Nesting and shorthands are great for authoring, but typically in `skin` we expand everything out into independent selectors because it makes them much easier to find via grep/document search

Reply:
Aww man. Is it okay to nest pseudo selectors?

Reply:
So we generally don't allow pseudo selectors nested like this, because when we are trying to debug issues, its really easy to look up these selectors when they are not nested. I can copy the selector from the inspector and see exactly where it matches in our code. With nested selectors, we don't get that. 

This might be changed if people are using source maps, but that doesn't always happen.